### PR TITLE
blockly start / first custom blocks and pritning to console when

### PR DESCRIPTION
**Pull request by paulsonnentag**

### CODE REVIEW

File: blockly/blocklyTest.html [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+      var workspace = Blockly.inject('blocklyDiv',
+        {toolbox: document.getElementById('toolbox')})
+
+      //workspace.addChangeListener(myUpdateFunction)
+
+      document.getElementById('run').onclick = function () {
+        myUpdateFunction()
+      }
+
+      Blockly.JavaScript['move'] = function(block) {
```
Comment (paulsonnentag) [inline blockly/blocklyTest.html]: I would use Blockly.Javascript.move instead

File: blockly/blocklyTest.html [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+      </category>
+      <sep></sep>
+      <category name="Movement" colour="210">
+        <block type="move"></block>
+        <block type="rotate"></block>
+      </category>
+    </xml>
+
+    <script>
+      var workspace = Blockly.inject('blocklyDiv',
```
Comment (paulsonnentag) [inline blockly/blocklyTest.html]: wrap the whole blockly editor as a choo component. Use lifecycle hooks for initialization

File: blockly/blocklyTest.html [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    <div id="blocklyDiv" style="height: 480px; width: 600px;"></div>
+    <textarea id="textarea"></textarea>
+    <button id="run">Run</button>
+
+    <script src="../node_modules/blockly/blockly_compressed.js"></script>
+    <script src="../node_modules//blockly/blocks_compressed.js"></script>
+    <script src="../node_modules/blockly/javascript_compressed.js"></script>
+    <script src="../node_modules/blockly/msg/js/en.js"></script>
+    <script src="./custom_blocks/custom.js"></script>
+
```
Comment (paulsonnentag) [inline blockly/blocklyTest.html]: extract xml in separate file

File: blockly/custom_blocks/library.xml [hunk 4]
```diff
@@ -0,0 +1 @@
+move AUTO BOTH LEFT Move Move Left LEFT Right RIGHT Up UP Down DOWN 240 rotate AUTO BOTH LEFT Rotate Direction Left LEFT Right RIGHT 330
```
Comment (paulsonnentag) [inline blockly/custom_blocks/library.xml]: Delete this file

**Review by timgrossmann — APPROVED** [review]

**Pull request closed without merging**